cmd/podman/utils.go

```diff
@@ -77,7 +77,7 @@ func startAttachCtr(ctr *libpod.Container, stdout, stderr, stdin *os.File, detac
 		return ctr.Attach(streams, detachKeys, resize)
 	}
 
-	attachChan, err := ctr.StartAndAttach(getContext(), streams, detachKeys, resize)
+	attachChan, err := ctr.StartAndAttach(getContext(&cli.Context{}), streams, detachKeys, resize)
```

Comment:
this doesn't seem correct.  `getContext` will then try to use `.Ctx` and it is nil causing a segfault

Reply:
I mean it wouldn't as the condition checks on the `trace` bool?

Reply:
Added extra nil check condition as requested to `getContext()`.